lxd/init.go

```diff
@@ -26,6 +26,10 @@ type initDataCluster struct {
 	// The path to the cluster certificate
 	// Example: /tmp/cluster.crt
 	ClusterCertificatePath string `json:"cluster_certificate_path" yaml:"cluster_certificate_path"`
+
+	// A cluster join token
+	// Example: BASE64-TOKEN
+	ClusterToken string `json:"cluster_token" yaml:"cluster_token"`
```

Comment:
ClusterJoinToken for consistency and to indicate its just for joining.

Reply:
Hmm, `ClusterAddress`, `ClusterCertificate`, `ClusterPassword` are all similarly used for join despite not having their name reflect it, so this felt more in line with the rest of them.

Reply:
My thinking was that those don't expire after joining though, and the comments all say join token but field name was slightly different. I see what you mean though.

Reply:
`ClusterAddress` kinda expires after joining as that gets updated with every hearbeat. `ClusterCertificate` expires the next time you use `lxc cluster update-certificate`, `ClusterPassword` as used in the past would expire almost immediately (admin unsetting it).

Reply:
Yeah im not dead set against it. I was mainly thinking from an integration point of view as our docs talk about a join token (and not join password for instance) and the field doesn't match that term, so might think its something else. I think most folks would take the join context into account and/or figure it out from the api docs themselves. I just thought I'd flag it as it triggered my inconsistency alarm 😀 happy either way.

Reply:
yeah, I'm not super keen on making the key even longer than it is. I'm actually kinda confused as to why all those keys have `cluster_` as a prefix when they're all already part of a cluster API... but not a lot we can do about that now.